README.md

```diff
@@ -32,45 +32,49 @@ Any versions not listed here _may_ work but they are not automatically tested.
 
 ## Usage
 
-Please refer to an example in [API documentation](https://docs.omise.co/) or the [help](https://docs.python.org/2/library/functions.html#help) function for documentation. For basic usage, you can use the module in your application by importing the `omise` module and set the secret key and public key:
+Please refer to examples in our [API documentation](https://www.omise.co/docs).
+For basic usage, you can use the package in your application by importing `omise` and setting the secret key:
 
 ```python
->>> import omise
->>> omise.api_secret = 'skey_test_4xsjvwfnvb2g0l81sjz'
->>> omise.api_public = 'pkey_test_4xs8breq32civvobx15'
+import omise
+omise.api_secret = 'skey_test_4xsjvwfnvb2g0l81sjz'
 ```
 
-After both keys are set, you can now use all the APIs. For example, to create a new customer without any cards associated to the customer:
+After the secret key is set, you can use all APIs which use secret key authentication.
+For example, to create a new customer without any cards associated to the customer:
 
 ```python
->>> customer = omise.Customer.create(
->>>    description='John Doe',
->>>    email='john.doe@example.com'
->>> )
+customer = omise.Customer.create(
+   description='John Doe',
+   email='john.doe@example.com'
+)
 <Customer id='cust_test_4xtrb759599jsxlhkrb' at 0x7ffab7136910>
 ```
```

Comment:
How about using `create Charge` example?
That would be good if it relates to other libs, like the example on Omise-GO, Omise-Ruby, Omise-Java, etc.
There use `create Charge` for an example
🙇

Reply:
Good suggestion!

Reply:
Done